fem/estimators.cpp

```diff
@@ -324,6 +295,10 @@ void KellyErrorEstimator::ComputeEstimates()
             error_estimates(FT->Elem2No) += jump_integral;
          }
       }
+      else     // We area t a boundary face
+      {
+         // TODO implement boundary flux difference
```

Comment:
Also, do you intend to address thee TODOs you added before merging? (also in line 245, 347) ?

Reply:
No, this PR is a bug-fix and I do not intend to add new features. However, I forgot to add the hints in the original PR, so I just wanted to leave them in this PR (since I felt like it is minor). I can remove the hints if you feel they are out of place.

Reply:
Having hints is better than not having hints. For me, it's perfectly fine. Thanks for clarifying!